more_itertools/recipes.py

```diff
@@ -1075,3 +1077,89 @@ def totient(n):
     for prime in set(factor(n)):
         n -= n // prime
     return n
+
+
+# Miller–Rabin primality test: Testing against small sets of bases
+# https://w.wiki/CGwr
+_perfect_tests = [
+    (2047, (2,)),
+    (9080191, (31, 73)),
+    (4_759_123_141, (2, 7, 61)),
```

Comment:
We should be consistent. Either use underscores in all of the large numbers of none of them.  I slight favor none-of-them.